src/NewCommand.php

```diff
@@ -289,8 +289,7 @@ protected function pushToGitHub(string $name, string $directory, InputInterface
         $branch = $input->getOption('branch') ?: $this->defaultBranch();
```

Comment:
Isn't this variable `$branch` (or even the whole line) useless now?

Reply:
I removed this line: https://github.com/laravel/installer/commit/ea0b3b6e5f9d6486bdf05342581224b007b817c0